share/org.getmonero.Monero.yaml

```diff
@@ -40,46 +36,46 @@ modules:
       - ./b2 -j$FLATPAK_BUILDER_N_JOBS install variant=release --layout=system
     sources:
       - type: archive
-        url: https://boostorg.jfrog.io/artifactory/main/release/1.82.0/source/boost_1_82_0.tar.gz
-        sha256: 66a469b6e608a51f8347236f4912e27dc5c60c60d7d53ae9bfe4683316c6f04c
+        url: https://archives.boost.io/release/1.84.0/source/boost_1_84_0.tar.gz
```

Comment:
https://github.com/monero-project/monero-gui/blob/c957058860194b7c6113811556d8e3405a08a03a/Dockerfile.linux#L159

Reply:
> https://github.com/monero-project/monero-gui/blob/c957058860194b7c6113811556d8e3405a08a03a/Dockerfile.linux#L159

The flatpak already uses [1.88.0](https://github.com/flathub/org.getmonero.Monero/blob/cedc82ede03ccade1ccf5871314f7045a0906e2b/org.getmonero.Monero.yaml#L71). Maybe I should use the newer version if there are no problems with it (not only about this case)?